src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -44,57 +46,58 @@
 
 ;; ACE creation
 
-(defn munge-name
-  [pattern]
-  (-> pattern (str/lower-case) (str/split #"\.") reverse vec))
+(schema/defn ^:private split-domain :- [String]
+  "Given a domain string, split it on '.' and reverse it. For examples,
+  'sylvia.plath.net' becomes ['net' 'plath' 'sylvia']. This is used for domain
+  matching."
+  [pattern :- String]
+  (-> pattern
+      (clojure.string/lower-case)
+      (clojure.string/split #"\.")
+      reverse
+      vec))
```

Comment:
I don't exactly remember the original purpose, but I think it performed way better as a vec than a sequence.